app/src/main/java/com/example/dine_and_donate/OldVouchersFragment.java

```diff
@@ -61,34 +66,43 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             @Override
             public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                 //look through all restaurants
-                for(DataSnapshot dsRestaurant : dataSnapshot.getChildren()) {
+                for (DataSnapshot dsRestaurant : dataSnapshot.getChildren()) {
                     //iterate through all events at that restaurant
-                    for(DataSnapshot dsEvent : dsRestaurant.getChildren()) {
+                    for (DataSnapshot dsEvent : dsRestaurant.getChildren()) {
                         //that event is saved, should be added to arrayList
-                        if(pastEvents.containsKey(dsEvent.getKey())) {
+                        if (pastEvents.containsKey(dsEvent.getKey())) {
                             //if event end date is older than today's date, it is a past event
-                            if(Long.toString(dateMillis).compareTo(dsEvent.child("endTime").toString()) > 0) {
-                                initBitmapsPastEvents(dsEvent.child("imageUrl").getValue().toString(), dsEvent.child("locationString").getValue().toString());
+                            if (Long.toString(dateMillis).compareTo(dsEvent.child("endTime").toString()) > 0) {
+                                initBitmapsPastEvents();
                             }
                         }
                     }
                 }
             }
+
             @Override
             public void onCancelled(@NonNull DatabaseError databaseError) {
 
             }
         });
         mRecyclerView = view.findViewById(R.id.rv_vouchers);
-        StaggeredRecyclerViewAdapter staggeredRecyclerViewAdapter = new StaggeredRecyclerViewAdapter(getActivity(), mNames, mImageUrls);
+        StaggeredRecyclerViewAdapter staggeredRecyclerViewAdapter = new StaggeredRecyclerViewAdapter(getActivity(), mEvents);
         StaggeredGridLayoutManager staggeredGridLayoutManager = new StaggeredGridLayoutManager(2, LinearLayoutManager.VERTICAL);
         mRecyclerView.setLayoutManager(staggeredGridLayoutManager);
         mRecyclerView.setAdapter(staggeredRecyclerViewAdapter);
     }
 
     //add images and descriptions to respective arrayLists
-    private void initBitmapsPastEvents(String mUrls, String mDescriptions) {
-        mImageUrls.add(mUrls);
-        mNames.add(mDescriptions);
+
+    private void initBitmapsPastEvents() {
```

Comment:
wait what's happening here? before you were getting events from the db and now you're just hardcoding them? is this a merge conflict issue?